src/components/Wallets/WalletMenuItem.vue

```diff
@@ -46,7 +46,6 @@ export default {
 .card.router-link-active {
   background: #f4f4f4;
   padding-bottom: 14px;
```

Comment:
Please change this padding (since it was placed there for a bottom line), otherwise it will be like this:
![sep-17-2018 10-44-42](https://user-images.githubusercontent.com/10958990/45611063-d226f880-ba66-11e8-8217-f3fadabb9e28.gif)

Reply:
However, @Pobepto will put it back in #57 :)